content/en/blog/2025/otel-js-sdk-2-0.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Announcing the OpenTelemetry JavaScript SDK 2.0
+linkTitle: OTel JS SDK 2.0
+date: 2025-03-21
+author: >
+  [Jamie Danielson](https://github.com/JamieDanielson) (Honeycomb)
+sig: OpenTelemetry JS
+cSpell:ignore: Danielson
+---
+
+Exciting news!
+[OpenTelemetry JavaScript](https://github.com/open-telemetry/opentelemetry-js)
+has released
+[SDK 2.0](https://github.com/open-telemetry/opentelemetry-js/releases)!
+
+There is a
+[migration guide](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md)
+in the repository detailing breaking changes and steps for users to take to
+upgrade to the latest versions.
+
+## What is JS SDK 2.x?
+
+"JS SDK 2.x" encompasses new releases of the `@opentelemetry/*` JavaScript
+packages published from the
+[`opentelemetry-js` repository](https://github.com/open-telemetry/opentelemetry-js),
+except the API and semantic-conventions packages. The package versions for this
+new major will be `>=2.0.0` for the stable and `>=0.200.0` for the unstable
+packages. Details on the full list of packages can be found in the
+[migration guide](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/upgrade-to-2.x.md#what-is-js-sdk-2x).
```

Comment:
If you commit the `{{% alert-md ... %}}` suggested previously, you'll be able to reuse the markdown link definition:

```suggestion
[migration guide][].
```